sonar-project.properties

```diff
@@ -0,0 +1 @@
+sonar.cpd.exclusions=src/generators/javascript/renderers/ClassRenderer.ts, src/generators/java/renderers/ClassRenderer.ts, src/generators/typescript/renderers/ClassRenderer.ts, src/generators/javascript/JavaScriptRenderer.ts, src/generators/typescript/renderers/EnumRenderer.ts, src/generators/java/renderers/EnumRenderer.ts
```

Comment:
Probably you can use wildcards here :)

Reply:
I disagree actually. 

I think we have to specifically say which files should have removed this check. Otherwise, we won't catch certain checks on files we didnt approve. If that makes sense 😄

Reply:
probably renderers will have always duplication for `renderProperties` etc :)

Reply:
Yea true, but then we must explicitly say it is 🙂 

I just wonder whether this config file should be applied now or not until it is merged 🤔 ?

Reply:
I don't know, but sonar is "old" solution, so probably after merging. Do you want to accept it?

Reply:
Please also add 
![image](https://user-images.githubusercontent.com/20404945/120221037-01451e80-c23e-11eb-9c13-1af9692e77a6.png)
files

Reply:
They are fixed by the new interpreter, so it solves it self.